lib/pdk/cli/update.rb

```diff
@@ -48,6 +48,18 @@ module PDK::CLI
 
       updater = PDK::Module::Update.new(opts)
 
+      if updater.pinned_to_puppetlabs_template_tag?
+        PDK.logger.info _(
```

Comment:
Potentially (not right now) the default template could be pointed to a different location via the PDK config.  Should this instead be about the `default` template URI instead of the `puppetlabs` template URI?